test/rules.test.js

```diff
@@ -0,0 +1,68 @@
+const latinVarNameOnly = require('../src/rules/latin-var-name-only');
+
+let RuleTester = require('eslint').RuleTester;
+
+let ruleTester = new RuleTester();	
+
+Object.assign(ruleTester.testerConfig, {
+	parserOptions: {
+		ecmaVersion:2018
+	}
+}); 
+// Проверить свойства объектов и аргументы функции, function declaration, named function expression
+		ruleTester.run('latin-var-name-only', latinVarNameOnly, {
+			valid: [{
+					code: 'var whiteAlbum = true'
+				},
+				{
+					code: 'var $whiteAlbum = true'
+				},
+				{
+					code: 'var white_Album = true'
+				},
+				{
+					code: 'let kraftWork = 7'
+				},
+				{
+					code: 'function rollingStones(){return 1;}',
+				},
+				{
+					code: 'let kraftWork = 7'
+				},
+				{
+					code:'[34,45].forEach((element) => element)',
+				},
+				{
+					code:'let obj = {zno:"the end"}',
+				}
+			],
+			invalid: [{
+					code: 'let табулатура = true',
+					errors: [{ message: "variable name consist non-latin char." }],
```

Comment:
Don't repeat yourself. If the error message is always the same define it in a constant